File: PizzaShopAdmin/ClientApp/src/dto/Pizza/PizzaDto.ts
import { PriceDto } from './PriceDto';
import { IngredientDto } from './IngredientDto';

export class PizzaDto {
    public id: number;
    public name: string;
    public description: string;
    public prices?: PriceDto[];
    public imgPath: string;
    public ingredients?: IngredientDto[];
}

File: PizzaShopAdmin/Dto/Pizza/PizzaDto.cs
using PizzaShop.EntityFramework.Entities;
using PizzaShopAdmin.Dto.Ingredient;
using System.Collections.Generic;

namespace PizzaShopAdmin.Dto.Pizza
{
    public class PizzaDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ImgPath { get; set; }
        public List<PriceDto> Prices { get; set; }
        public List<IngredientDto> Ingredients { get; set; }
    }
}
